### PR TITLE
Add new OQL String functions

### PR DESCRIPTION
**Pull request by dahfjkg**
Needs an internal review.
To be added in 9.21.0 release.

### CODE REVIEW

File: content/en/docs/refguide/modeling/resources/data-sets/oql/oql-expressions/oql-functions/oql-lower.md [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+
+## 2 Syntax
+
+The syntax is as follows:
+
+```sql
+LOWER ( expression )
+```
+
+`expression` specifies the string to which the transformation is to be applied. The value can be an attribute, a constant, or an OQL parameter.
```
Comment (basdebakker) [inline content/en/docs/refguide/modeling/resources/data-sets/oql/oql-expressions/oql-functions/oql-lower.md]: This is not correct. Attributes, constants and parameters are examples of expressions, but the expression can be anything. The value can only be a string.

**Review by basdebakker — APPROVED** [review]
Approved by data storage team.

**Comment by NatasaKralj** [comment]
Language reviewed and edited. PR is ready for 9.21.0 release.

**Comment by NatasaKralj** [comment]
Added new functions to list of reserved OQL words

**Comment by basdebakker** [comment]
> Added new functions to list of reserved OQL words

The new functions are not reserved words. We took care of that in the implementation to avoid invalidating older OQL that may use these.

**Comment by NatasaKralj** [comment]
> > Added new functions to list of reserved OQL words
> 
> The new functions are not reserved words. We took care of that in the implementation to avoid invalidating older OQL that may use these.

Oh, my mistake then. Thank you for letting me know. I'll revert the reserved words list.

**Review by NatasaKralj — APPROVED** [review]
Ready for 9.21 and approved

**Pull request merged**